config/state_convert.go

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"github.com/clearlinux/mixer-tools/helpers"
+)
+
+// CurrentStateVersion is the current revision for the state file structure
+const CurrentStateVersion = "1.0"
+
+func (state *MixState) parseVersionAndConvert() error {
+	// Reset version for files without versioning
+	state.version = "0.0"
```

Comment:
This is not needed for mixer.state because unlike builder.conf, it's file always had the `#VERSION` header, so there is no valid mixer.state file without versioning.